lang/julia/julia.talon

```diff
@@ -0,0 +1,30 @@
+tag: user.julia
+-
+tag(): user.code_imperative
+
+tag(): user.code_comment_line
+tag(): user.code_data_bool
+tag(): user.code_data_null
+tag(): user.code_functions
+tag(): user.code_functions_common
+tag(): user.code_keywords
+tag(): user.code_operators_assignment
+tag(): user.code_operators_array
+tag(): user.code_operators_math
+
+settings():
+    user.code_private_function_formatter = "SNAKE_CASE"
+    user.code_public_function_formatter = "SNAKE_CASE"
+    user.code_private_variable_formatter = "SNAKE_CASE"
+    user.code_protected_variable_formatter = "SNAKE_CASE"
+    user.code_public_variable_formatter = "SNAKE_CASE"
+
+[state] [at] assert: "@assert "
```

Comment:
It's probably worth making the state prefix mandatory for consistency with other languages.

Also are the dassert things standard Julia? I couldn't find them with a web search.